### PR TITLE
Editorial: ToUint8Clamp guard against infinity doubles

### PR DESCRIPTION
**Pull request by dewren99**
Fixes https://github.com/tc39/ecma262/issues/2395

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -5220,6 +5220,7 @@ <h1>
       </dl>
       <emu-alg>
         1. Let _number_ be ? ToNumber(_argument_).
+        1. If _number_ is either *+∞*<sub>𝔽</sub> or *-∞*<sub>𝔽</sub>, return *255*<sub>𝔽</sub>.
```
Comment (ljharb) [inline spec.html:5223]: shouldn't -∞ clamp to 0?

Comment (dewren99) [reply spec.html:5223]: I was following the suggestion from @michaelficarra , but it also makes sense to me that "-∞ would be <= 0 and so it'd return 0."

Comment (michaelficarra) [reply spec.html:5223]: Yeah this was my bad, we should return 0 for -Infinity.

Comment (ljharb) [reply spec.html:5223]:
```suggestion
        1. If _number_ is *+∞*<sub>𝔽</sub>, return *255*<sub>𝔽</sub>.
        1. If _number_ is *-∞*<sub>𝔽</sub>, return *0*<sub>𝔽</sub>.
```

**Comment by ljharb** [comment]
This might be a duplicate of #3083.

**Review by michaelficarra — APPROVED** [review]

**Comment by michaelficarra** [comment]
@ljharb We should land this first, since this involves correctness ("normative-editorial").

**Comment by bakkot** [comment]
@michaelficarra We call bugfixes normative when the previous spec was coherent, just not matching intent. Here the previous spec wasn't coherent so it's definitely editorial. Still happy to land it first though.

**Review by bakkot — APPROVED** [review]

**Pull request merged**